core-bundle/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "scripts": {
+        "dev": "NODE_ENV=development node_modules/.bin/encore dev --watch",
+        "prod": "env NODE_ENV=production node_modules/.bin/encore production"
```

Comment:
Might be a stupid question but why does `prod` need an `env` command and `dev` does not?

Reply:
no idea, I copied that from my local recipes. But it seems unnecessary, I've added the defaults in 63b6b625e2be22927f66253bef099762fbd2075b